rmgpy/data/kinetics/family.py

```diff
@@ -1189,8 +1200,14 @@ def add_rules_from_training(self, thermo_database=None, train_indices=None):
             if type(data) is Arrhenius:
                 # more specific than isinstance(data,Arrhenius) because we want to exclude inherited subclasses!
                 data = data.to_arrhenius_ep()
-            elif isinstance(data, StickingCoefficient):
-                data = StickingCoefficientBEP(
+            elif isinstance(data, SurfaceArrhenius):
+                if has_sticking_coeff:
+                    if surface_site_density is None:
+                        surface_site_density = 2.483e-09
```

Comment:
I think instead of having this hard coded in, you should load it from the metal database

Reply:
don't remember what the units are in the db, so this might affect your reaction.py additions